test/local/tokens/account_reset_token.js

```diff
@@ -51,7 +51,7 @@ describe('account reset tokens', () => {
           (x) => {
             token = x
             assert.equal(token.data.toString('hex'), tokenData)
-            assert.equal(token.id.toString('hex'), '46ec557e56e531a058620e9344ca9c75afac0d0bcbdd6f8c3c2f36055d9540cf')
```

Comment:
Given that `token.id` has always been a hex string, was this test previously in error?  It seems suspicious that it would previously have been a buffer.

Reply:
Oh wait, I think I see, it was always a hex string and the `toString()` was a no-op:

```
> "hello".toString("hex")
'hello'
```

Reply:
> it was always a hex string and the `toString()` was a no-op

Yep, this.